extensions/windows-event-log/Bookmark.cpp

```diff
@@ -11,7 +11,7 @@ namespace nifi {
 namespace minifi {
 namespace processors {
 
-Bookmark::Bookmark(const std::wstring& channel, const std::wstring& query, const std::string& bookmarkRootDir, const std::string& uuid, std::shared_ptr<logging::Logger> logger)
+Bookmark::Bookmark(const std::wstring& channel, const std::wstring& query, const std::string& bookmarkRootDir, const std::string& uuid, bool processOldEvents, std::shared_ptr<logging::Logger> logger)
```

Comment:
This is an incompatible API change.
Why don't you put the bool to the end with a default value?

Reply:
The change in Bookmark is internal to CWEL, it doesn't affect other code, it is not necessary to maintain API signature.